encoder/vaapiencoder_h264.cpp

```diff
@@ -700,13 +700,14 @@ class VaapiEncoderH264Ref
     SurfacePtr m_pic;
 };
 
-VaapiEncoderH264::VaapiEncoderH264():
-    m_numBFrames(0),
-    m_reorderState(VAAPI_ENC_REORD_WAIT_FRAMES),
-    m_streamFormat(AVC_STREAM_FORMAT_ANNEXB),
-    m_frameIndex(0),
-    m_keyPeriod(30),
-    m_idrNum(0)
+VaapiEncoderH264::VaapiEncoderH264()
```

Comment:
I always use command like this `~/bin/git-clang-format --style=WebKit --binary=clang-format-3.7 HEAD^`,  then `git ci -a --amend`.